src/sentry/search/utils.py

```diff
@@ -56,14 +57,22 @@ def get_user_tag(projects: Sequence[Project], key: str, value: str) -> str:
     return euser.tag_value  # type: ignore
 
 
-def parse_status_value(value: Union[str, int]) -> int:
-    if value in STATUS_QUERY_CHOICES:
-        return int(STATUS_QUERY_CHOICES[value])
-    if value in STATUS_QUERY_CHOICES.values():
-        return int(value)
+def parse_status_value(status: Union[str, int]) -> int:
+    if status in STATUS_QUERY_CHOICES.keys():
```

Comment:
Better off not using `.keys` here, this changes this from an `O(1)` operation to `O(N)`